docs/scRNAseq_tutorials.md

```diff
@@ -0,0 +1,25 @@
+## Become more familiar with scRNAseq analysis thanks to these tutorials
+
+Here are some recommandations on learning how to perform scRNAseq analysis. You can find some tutorials on Galaxy or on R.
```

Comment:
recommendations ?

Reply:
You can find some tutorials on Galaxy or on R. : WHERE ?

Reply:
@bellenger-l  je te montrerai comment mettre des liens à des balises intra-documents. (mais tu dois pouvoir trouver seule dans les manuels de markdown)

Reply:
Je voulais dire des tuto à faire soit sur galaxy soit en R. J'avais mis `in` à la base mais Leo a dit que `on` était plus approprié. Ce n'est peut être toujours pas la bonne formulation

Reply:
Dans ce cas, je dirais:
You can find some tutorials, that can be run under the Galaxy (url)  or R (url) environment.

Reply:
Ok mais je comprends pas très bien quels sont les urls à mettre puisque je les liste ensuite. 

> You can find some tutorials, that can be run under the Galaxy (url) or R (url) environment (find the different urls below).